classes/doi/exceptions/DoiException.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+/**
+ * @file classes/doi/exceptions/DoiException.php
+ *
+ * Copyright (c) 2014-2023 Simon Fraser University
+ * Copyright (c) 2000-2023 John Willinsky
+ * Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
+ *
+ * @class DoiException
+ *
+ * @brief Base class to catch for all DOI-related exceptions
+ */
+
+namespace PKP\doi\exceptions;
+
+class DoiException extends \Exception
```

Comment:
Hmmm... Why do we need this empty class?

Reply:
The idea was to create a base class for DOI-related exceptions, so they could be caught, but looking again, I can refactor the existing `DoiActionException` to work in all cases so we won't need an empty base class.